tests/test_event_service.py

```
@@ -1,114 +1,75 @@
         result = event_service.get_all_events()
 
         assert result == []
 
-    @patch("application.services.event_service.get")
+    @patch("builtins.open", new_callable=mock_open, read_data="irrelevant")
+    @patch("yaml.safe_load", side_effect=Exception("File error"))
     @patch("application.services.event_service.logger")
-    def test_get_all_events_connection_error(
-        self, mock_logger, mock_get, event_service
+    def test_get_all_events_file_error(
+        self, mock_logger, mock_yaml_load, mock_file, event_service
```

Ruff (0.12.2)

Unused method argument: `mock_yaml_load`

(ARG002)